### ISSUE TITLE
if you have tested the changes in the use of VRAM for graphics card rendering before and after compression?

### ISSUE THREAD

**Issue by smart4654154** [YihangChen-ee/HAC#13]
Thank you very much for your work
The data compression work is excellent
May I ask if you have tested the changes in the use of VRAM for graphics card rendering before and after compression?

**Comment by YihangChen-ee** [comment]
Hi, the rendering memory would not change before or after compression. HAC is a pre-decoding technique, which is used to decode the model weights (ie, the Scaffold-GS model) given the input bitstreams. Once the weights are obtained after decoding, there will be nothing to do with HAC. So the rendering process is totally the same as Scaffold-GS, with no extra time or memory consumption.

**Comment by smart4654154** [comment]
thank
Are you saying that you have reduced the disk usage of the result files, but the space for data in VRAM and the VRAM used for rendering have not been reduced?

**Comment by YihangChen-ee** [comment]
Yes. We are doing storage compression, but not doing rendering efficiency. Storage compression can help with smaller storage usage and shorter transmission time.
